androidApp/src/main/java/org/datepollsystems/waiterrobot/android/MainActivity.kt

```diff
@@ -38,12 +38,12 @@ class MainActivity : AppCompatActivity() {
         // Switch from SplashScreenTheme to AppTheme
         setTheme(R.style.Theme_WaiterRobot_Main)
 
-        checkForUpdate()
-
         setContent {
             RootView(vm, ::onAppThemeChange)
         }
 
+        checkForUpdate()
```

Comment:
If there is a reason for this to be after the `setContent` we should add a comment perhaps to explain this a little bit further

Reply:
No reason - I think that was just moved when I tried something and then did not revert it :)